docs/guide.pug

```diff
@@ -325,7 +323,7 @@ block content
     define a `fullName` property that won't get persisted to MongoDB.
 
     ```javascript
-    personSchema.virtual('fullName').get(function () {
+    personSchema.virtual('fullName').get(() => {
```

Comment:
This change doesn't work because the virtual getter uses `this`, which [arrow functions](https://masteringjs.io/tutorials/fundamentals/arrow) mess up.